demo/web/index.html

```diff
@@ -32,7 +32,7 @@
       async function startRhino(accessKey) {
         try {
           writeMessage("Rhino is loading. Please wait...");
-          let rhino = await RhinoWeb.RhinoWorker.create(
+          window.rhino = await RhinoWeb.RhinoWorker.create(
```

Comment:
why does rhino need to be in the window scope?

Reply:
Good point, it does not